assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { noop } from 'lodash';
+import { format, parseISO } from 'date-fns';
+
+import Banner from '@common/Banners/Banner';
+import Button from '@common/Button';
+import Modal from '@common/Modal';
+import PageHeader from '@common/PageHeader';
+import Table from '@common/Table';
+import Tooltip from '@common/Tooltip';
+
+const defaultUsers = [];
+
+function Users({
+  onDeleteUser = noop,
+  navigate = noop,
+  users = defaultUsers,
+  loading = true,
```

Comment:
I would put the default value for loading to `false`. 
You just manage the real value in the `UsersPage`.
Otherwise you are forced to put `loading: false` in all storybook entries